pallets/ddc-verification/src/aggregator_client.rs

```diff
@@ -646,6 +646,15 @@ pub(crate) mod json {
 		pub number_of_gets: u64,
 	}
 
+	impl EHDUsage {
+		pub fn has_usage(&self) -> bool {
+			self.stored_bytes > 0 &&
```

Comment:
Maybe add `#[derive(Default)]` to `EHDUsage` and compare with the default value here?